.env.example

```
@@ -0,0 +1,2 @@
+ACME_CA_SERVER="https://acme.proeftuin.uzi-online.rdobeheer.nl/"
+YUBIKEY_PIN="123456"
```

```suggestion
OIDC_PROVIDER_BASE_URL="https://proeftuin.uzi-online.rdobeheer.nl"
ACME_CA_SERVER="https://acme.proeftuin.uzi-online.rdobeheer.nl"
YUBIKEY_PIN="123456"
```

But maybe we now should add an trailing slash check/ ensure trailing slash function so that it does not matter in the url includes a slash or not.

---

This has now been resolved by using the `urllib.parse.ParseResult` in combination with `urljoin`.